inscriptions/ISic003659.xml

```diff
@@ -191,76 +196,32 @@
-            <div type="edition" xml:space="preserve" subtype="EDR" n="EDR072804" xml:lang="la">
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#MB">
                 <ab>
-                    <lb n="1"/>
-                    XVII
-                    <expan>
-                        <abbr>K</abbr>
-                        <ex>alendas</ex>
-                    </expan>
-                    <expan>
-                        <abbr>Septemb</abbr>
-                        <ex>res</ex>
-                    </expan>
-                    <lb n="2"/>Feridius Cereris dominae
-                    <supplied reason="lost">
-                        <expan>
-                            <abbr>s</abbr>
-                            <ex>ervus</ex>
-                        </expan>
-                    </supplied>
-                    <lb n="3"/>hic sibi
-                    <choice>
-                        <sic>suabiter</sic>
-                        <corr>suaviter</corr>
-                    </choice>
-                    <expan>
-                        <abbr>fec</abbr>
-                        <ex>it</ex>
-                    </expan>
-                    e
-                    <supplied reason="lost">o</supplied>
-                    <lb n="4" break="no"/>rum tres adulescentes<lb n="5"/>quorum nomina lege<lb n="6"/>Onesimus et
-                    <expan>
-                        <abbr>L</abbr>
-                        <ex>ucius</ex>
-                    </expan>
-                    Valerius<lb n="7"/>Cas
-                    <expan>
-                        <ex>s</ex>
-                        <abbr>ianus</abbr>
-                    </expan>
-                    et Filumenus<lb n="8"/>unus cum
-                    <choice>
-                        <sic>mulierae</sic>
-                        <corr>muliere</corr>
-                    </choice>
-                    ea<lb n="9"/>Taurus multis annis
-                    <choice>
-                        <sic>habe</sic>
-                        <corr>have</corr>
-                    </choice>
-                    <expan>
-                        <abbr>facian</abbr>
-                        <ex>t</ex>
-                    </expan>
-                    <lb n="10"/>co
-                    <expan>
-                        <ex>n</ex>
-                        <abbr>iu</abbr>
-                        <ex>n</ex>
-                        <abbr>ximus</abbr>
-                    </expan>
-                    <lb n="11"/>propitiam.
+                    <lb n="1"/><num value="17">XVII</num> <expan><abbr>K</abbr><ex>alendas</ex></expan>
+                    <expan><abbr>Septemb</abbr><ex>res</ex></expan>
+                    <lb n="2"/><persName><name>Feridius</name></persName> <name>Cereris</name> dominae
+                    <supplied reason="lost"><expan><abbr>s</abbr><ex>ervus</ex></expan></supplied>
+                    <lb n="3"/>hic sibi <choice><sic>suabiter</sic><corr>suaviter</corr></choice>
```

Comment:
not sic and corr (because that implies a mistake) but rather reg and orig (because it is a phonetic variant)